spoon-runner/src/main/java/com/squareup/spoon/SpoonClearingAndroidTestRunner.java

```diff
@@ -0,0 +1,77 @@
+package com.squareup.spoon;
+
+import com.android.ddmlib.AdbCommandRejectedException;
+import com.android.ddmlib.CollectingOutputReceiver;
+import com.android.ddmlib.IShellEnabledDevice;
+import com.android.ddmlib.Log;
+import com.android.ddmlib.ShellCommandUnresponsiveException;
+import com.android.ddmlib.TimeoutException;
+import com.android.ddmlib.testrunner.ITestRunListener;
+import com.android.ddmlib.testrunner.RemoteAndroidTestRunner;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Run the "pm clear" command before every `run` invocation.
+ */
+public class SpoonClearingAndroidTestRunner extends RemoteAndroidTestRunner {
+
+    private static final String LOG_TAG = "SpoonClearingAndroidTestRunner";
+    private IShellEnabledDevice mRemoteDevice;
+    private CollectingOutputReceiver mOutputReceiver;
+    private long mMaxTimeoutMs = 10000L;
```

Comment:
`mMaxTimeoutMs` and `mMaxTimeToOutputResponseMs` should both just be static constants

Reply:
Also delete all the 'm' prefixes.